lib/ht_sip_validator/sip/sip.rb

```diff
@@ -68,6 +78,15 @@ def extract
       end
     end
 
+    # @return [Array] sub-set of filenames in a given group
+    def group_files(group)
```

Comment:
The name of this function might be misleading in that it could be taken to mean the imperative, "take these files and group them."  However, alternatives seem verbose.  
```
files_from_group(group)
files_of_group(group)
files_of(group) 
files_from(group)
```
@malakai97 any suggestions for naming?